### PR TITLE
[17.0][MIG] stock_picking_return_restricted_qty: Migration to 17.0

### PR DESCRIPTION
**Pull request by ced-adhoc**

### CODE REVIEW

**Review by luc-adhoc — APPROVED** [review]
functional review

**Review by mav-adhoc — APPROVED** [review]

File: stock_picking_return_restricted_qty/models/stock_picking_type.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+class StockPickingType(models.Model):
+    _inherit = "stock.picking.type"
+
+    restrict_return_qty = fields.Boolean(
+        string="Restrict Return Quantity",
+        help=(
+            "Enable this option to restrict returning more quantities "
+            "than delivered."
+        ),
+        default=False,
```
Comment (rousseldenis) [inline stock_picking_return_restricted_qty/models/stock_picking_type.py]: This is the default value

File: stock_picking_return_restricted_qty/tests/test_stock_picking_return_restricted_qty.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+from odoo.tests import Form, common
+
+
+class StockPickingReturnRestrictedQtyTest(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        # Remove this variable in v16 and put instead:
+        # from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
+        DISABLED_MAIL_CONTEXT = {
```
Comment (rousseldenis) [inline stock_picking_return_restricted_qty/tests/test_stock_picking_return_restricted_qty.py]: You can inherit from `BaseCommon` class instead. See https://github.com/odoo/odoo/blob/17.0/odoo/addons/base/tests/common.py#L21

File: stock_picking_return_restricted_qty/models/stock_picking_type.py (hunk 1 above)
Comment (ced-adhoc) [reply stock_picking_return_restricted_qty/models/stock_picking_type.py]: True, thanks

File: stock_picking_return_restricted_qty/tests/test_stock_picking_return_restricted_qty.py (hunk 2 above)
Comment (ced-adhoc) [reply stock_picking_return_restricted_qty/tests/test_stock_picking_return_restricted_qty.py]: fixed, thanks

**Comment by mav-adhoc** [comment]
LGTM

**Review by rousseldenis — APPROVED** [review]
LGTM

**Review by jcadhoc — APPROVED** [review]

**Comment by ced-adhoc** [comment]
Hi @rousseldenis , could you please merge? thanks!

**Pull request merged**